config/app/envs/trial.yaml

```diff
@@ -22,36 +22,29 @@ console:
         spec:
           containers:
           - env:
-            - name: KIE_ADMIN_USER
-              value: adminUser
             - name: KIE_ADMIN_PWD
               value: RedHat
-            - name: KIE_MBEANS
-              value: enabled
-            - name: KIE_SERVER_CONTROLLER_USER
-              value: controllerUser
             - name: KIE_SERVER_CONTROLLER_PWD
               value: RedHat
-            - name: KIE_SERVER_USER
-              value: executionUser
             - name: KIE_SERVER_PWD
               value: RedHat
             - name: KIE_MAVEN_USER
               value: mavenUser
             - name: KIE_MAVEN_PWD
               value: RedHat
-            - name: PROBE_IMPL
-              value: probe.eap.jolokia.EapProbe
-            - name: PROBE_DISABLE_BOOT_ERRORS_CHECK
-              value: "true"
-            image: rhpam70-businesscentral-openshift
+            image: rhpam{{.Version}}-businesscentral-openshift
             imagePullPolicy: Always
+            resources:
+              requests:
+                memory: 768Mi
+              limits:
+                memory: 1Gi
             livenessProbe:
               exec:
                 command:
                 - /bin/bash
                 - -c
-                - curl --fail --silent -u adminUser:RedHat http://localhost:8080/kie-wb.jsp
```

Comment:
hmm, does the user setting their own password cause this probe to fail?

Reply:
It does cause it to fail, which is why I changed the probes to leverage the env vars directly instead.